config/routes.rb

```diff
@@ -3,7 +3,12 @@
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
+  
+  get 'home/index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.